packages/ckeditor5-table/tests/ui/colorinputview.js

```diff
@@ -302,12 +293,11 @@ describe( 'ColorInputView', () => {
 				sinon.assert.calledWithExactly( spy.lastCall, 'input' );
 			} );
 
-			it.skip( 'should have #selectedColor bound to the #value', () => {
+			it( 'should set #selectedColor to the #value upon dropdown opening', () => {
```

Comment:
Maybe we could also create test which tests the logic of `#selectedColor` when dropdown is closed, then it should not be bound.